### ISSUE TITLE
Very Low Contrast Issues for Improved Website Accessibility: Non Compilance of WCAG 2.1 Standards

### ISSUE THREAD

**Issue by barchakuz** [camicroscope/caMicroscope#859]
**Description**
There are instances of very low contrast between text and background colors on the website. This can make it challenging for users, especially those with low vision, to read and comprehend the content effectively.
**Why It Matters**
Adequate contrast of text is necessary for all users, especially users with low vision.

**Possible Solution**
Increase the contrast between the foreground (text) color and the background color. Large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.

**To Reproduce**
- Install Wave Testing Extension from chrome store
- Navigate to the specified page on the website.
- Use the WAVE extension to scan the webpage for accessibility issues.
- See error

**Expected behavior**
Text should have sufficient contrast against its background color to ensure readability for all users, particularly those with visual impairments. Increasing the contrast between text and background colors will enhance readability and accessibility.

**Screenshots**
All Contrast Errors are marked on each tested page
_Home Page_
![1](https://github.com/camicroscope/caMicroscope/assets/66128370/d67bf951-5aea-43ed-ad46-7f48f60f23fe)
_Slide Page_
![2](https://github.com/camicroscope/caMicroscope/assets/66128370/5f9f5071-6eea-4fb4-8b42-712ddf5e4ecd)
_Info Page_
![3](https://github.com/camicroscope/caMicroscope/assets/66128370/587dcf79-bdf4-4961-81ce-3fde9657f374)
_SignUp Page_
![4](https://github.com/camicroscope/caMicroscope/assets/66128370/a6e43968-9db7-4997-a71c-1d6f9c238622)

**Desktop:**
- OS: Win 10
- Browserchrome
- Version 2 2

**Additional context**
This issue violates WCAG 2.1 standards related to contrast (minimum) (1.4.3, Level AA). Addressing contrast errors is crucial for improving the accessibility of the website and ensuring an inclusive user experience for all visitors.

**Comment by ujjwaldubey1** [comment]
@barchakuz Is this issue is still workable or not ?

**Comment by birm** [comment]
@ujjwaldubey1 I'm closing this one since it's mentioned in a merged PR. I think you'd want to run a report using wave or something and use that as a guide for another PR or issue for any lingering issues.
